codalab/rest/bundles.py

```diff
@@ -224,12 +224,16 @@ def _create_bundles():
       "uploading" regardless of the bundle type, or 0 otherwise. Used when
       copying bundles from another CodaLab instance, this prevents these new
       bundles from being executed by the BundleManager. Default is 0.
+    - `avatar`: 1 if the bundle is the user's uploaded avatar,
```

Comment:
Should we replace this parameter with something more general, such as `parent_worksheet_required`, instead?

Reply:
I think that's a good idea. Update the parameter to be `parent_worksheet_required`